build/gulpfile.mixin.js

```diff
@@ -24,10 +24,17 @@ gulp.task('mixin', function () {
 		return;
 	}
 
-  // {{SQL CARBON EDIT}}
+	// {{SQL CARBON EDIT}}
+	let serviceUrl = 'https://raw.githubusercontent.com/Microsoft/sqlopsstudio/release/extensions/extensionsGallery.json';
+	if (quality) {
```

Comment:
You can assume you have quality by this point, since the method would have returned already if it wasn't defined.

Reply:
if it's the official release, what would be the value here? 

---
In reply to: [174278044](https://github.com/Microsoft/sqlopsstudio/pull/902#discussion_r174278044) [](ancestors = 174278044)

Reply:
Values are "stable" for the normal releases, and "insider" if we ever do daily builds out of master.

Reply:
ok then I'll change it to be default for stable 

---
In reply to: [174278863](https://github.com/Microsoft/sqlopsstudio/pull/902#discussion_r174278863) [](ancestors = 174278863)